### DISCUSSION TITLE
Welcome to UWL-Campus-Safety Discussions!

### DISCUSSION THREAD

**Discussion by M4rcusBC** [M4rcusBC/UWL-Campus-Safety#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together! This discussion is actively moderated by both humans and bots 🤖.

  To get started, comment below with an introduction of yourself and how you plan to contribute to the project.

**Comment by MetaBadgerData** [comment]
Hi! My name is Alex and I plan to contribute to this app in many different ways. Looking forward to hearing all of your thoughts and ideas to improve this project!
